backend-project/small_eod/users/tests/test_viewsets.py

```diff
@@ -9,6 +9,7 @@ class UserViewSetTestCase(ReadOnlyViewSetMixin, TestCase):
     basename = "user"
     serializer_class = UserSerializer
     factory_class = UserFactory
+    queries_less_than_limit = 5
```

Comment:
Czy to szczególny przypadek? Wydaje mi się, że domyślna wartość w tym miejscu powinna wystarczyć.